### ISSUE TITLE
import radical.pilot fails for broken `resource_osg.json`

### ISSUE THREAD

**Issue by jhprinz** [radical-cybertools/radical.pilot#1254]
We installed the current (maybe outdated) version from radical.pilot using pip today.

```
pip install radical.pilot
```

After that we could not 

```
import radical.pilot as rp
```

with complaining that the `resource_osg.json` is somehow not parseable. We renamed the file so that it is not recognized as a config file and then it worked. Is it possible that there is still a broken `.json` file in the distributed version?

**Comment by marksantcroos** [comment]
Thanks for reporting. We actually released yesterday, and it seems there was a last minute change made just before the release to that config, so an error might have slept in. We'll look into it.

**Comment by andre-merzky** [comment]
Hmm, I am afraid I can't reproduce this.  In a fresh virtualenv, I see (shortened):

```
$ cd /tmp/
$ virtualenv ve
Running virtualenv with interpreter /usr/bin/python2
New python executable in /tmp/ve/bin/python2
Also creating executable in /tmp/ve/bin/python
Installing setuptools, pkg_resources, pip, wheel...done.

$ source ve/bin/activate
$ pip install radical.pilot
Collecting radical.pilot
[...]
Successfully installed apache-libcloud-1.5.0 colorama-0.3.7 netifaces-0.10.4 ntplib-0.3.3 pymongo-2.8 python-hostlist-1.17 pyzmq-16.0.2 radical.pilot-0.45 radical.utils-0.45 saga-python-0.45 setproctitle-1.1.10

$ python -c 'import radical.pilot as rp; s=rp.Session(); print rp.version_detail'
new session: [rp.session.thinkie.merzky.017226.0001]                           \
database   : [mongodb://144.76.72.175/am]                                     ok
0.45

$ ./ve/share/radical.pilot/examples/00_getting_started.py 

================================================================================
 Getting Started (RP version 0.45)                                              
================================================================================
new session: [rp.session.thinkie.merzky.017226.0003]                           \
database   : [mongodb://144.76.72.175/am]                                     ok
[...]
closing session rp.session.thinkie.merzky.017226.0003                          \
close pilot manager                                                            \
wait for 1 pilot(s) *                                                         ok
close unit manager                                                            ok
session lifetime: 44.2s                                                       ok
--------------------------------------------------------------------------------
```

Can you please send the stack trace, and the output of the `radical-stack` command?

**Comment by jhprinz** [comment]
Hmmm, we had two people with the same error. I can check tomorrow again. Not sure if it had to do with them both installing radical.pilot before not inside a VE?

**Comment by andre-merzky** [comment]
Thanks Jan, please do so.  

If RP is also installed in `~/.local` or in system space, things might indeed look different -- but I would still be surprised about an invalid osg config file, which should not have shipped in any version, really.  The `radical-stack` output might help with checking if the stack is consistent.

**Comment by ibethune** [comment]
I also see this.  I set up a clean VE, installed 0.45 from pip.  I don't have RP anywhere in my system python setup.

```
(test_env) Iains-MacBook-Pro:examples ibethune$ radical-stack
python            : 2.7.13
virtualenv        : /Users/ibethune/Desktop/test_env
radical.utils     : 0.45
saga-python       : 0.45
radical.pilot     : 0.45
(test_env) Iains-MacBook-Pro:examples ibethune$ ./00_getting_started.py epsrc.archer
2017-03-02 21:28:07,910: radical.pilot       : MainProcess                     : MainThread     : ERROR   : skip config file /Users/ibethune/Desktop/test_env/lib/python2.7/site-packages/radical/pilot/configs/resource_osg.json: Couldn't parse resource configuration file '/Users/ibethune/Desktop/test_env/lib/python2.7/site-packages/radical/pilot/configs/resource_osg.json': No JSON object could be decoded.
```

I think we broke something by commenting out that config just prior to the release :(

**Comment by marksantcroos** [comment]
Andre, you have `~/.radical/` overrides for the osg file maybe?

Anyway, I guess you want to backport the config disabling and do a hotfix.

**Comment by andre-merzky** [comment]
```sh
$ l -d .local/lib/python2.7 
lrwxrwxrwx 1 merzky merzky 9 Jan 10 21:03 .local/lib/python2.7 -> /dev/null
```

;-)  

But, whatever the reason, a hotfix seems appropriate.  Iain, ok if I backport #1251 and re-release as `v0.45.1`?

**Comment by ibethune** [comment]
Agreed.

**Comment by ibethune** [comment]
@jhprinz this should now be fixed in the 0.45.1 release - please upgrade and close the ticket if it's working for you.
